recipes/AISHELL-1/NAR-ASR/NAR_BERT_ASR/README.md

```diff
@@ -0,0 +1,56 @@
+# AISHELL-1 Non-Autoregressive ASR with [NAR-BERT-ASR](https://arxiv.org/abs/2104.04805) models.
+
+
+# Introduction
+This folder contains recipes for [NAR-BERT-ASR](https://arxiv.org/abs/2104.04805) with [AISHELL-1](https://www.openslr.org/33/), a 150-hour Chinese ASR dataset.
+
+
+# How to run
+Step 1: Install extra dependencies.
+```
+pip install transformers
+```
+
+Step 2: Pretrain the NAR-BERT-ASR-Encoder.
+```
+python pretrain_encoder.py hparams/pretrain_encoder.yaml --data_folder=/localscratch/aishell/
+```
+
+Step 3: Fine-tune the NAR-BERT-ASR.
+```
+python train_nar_bert_asr.py hparams/train_nar_bert_asr.yaml --data_folder=/localscratch/aishell/ --pretrain_weights_dir=results/Pretrain_Encoder/save/{CKPT+XXXX-XX-XX+XX-XX-XX+XX}
+```
+
+
+# Performance summary
+Results are reported in terms of Character Error Rate (CER).
+
+| hyperparams file | Test CER | Dev CER | GPUs |
+|:--------------------------:|:-----:|:-----:|:-----:|
+| pretrain_encoder.yaml | 7.94 | 6.95 | 1xRTX 2080 Ti 11GB |
+| train_nar_bert_asr.yaml | 5.26 | 4.64 | 1xTITAN RTX 24GB |
+
+You can checkout our results (models, training logs, etc,) here:
+https://drive.google.com/drive/folders/1zdUUp-hdbpVvpV3jUCgBqet04HjqZfRy?usp=sharing
```

Comment:
Do you want us to host it on our side ? This is better for maintaining the recipe on the long-term (just in case your account stops or any other issue)